### PR TITLE
MultiMystery Utilities

### PR DESCRIPTION
**Pull request by Berserker66**
Feel free to just take parts of this.
This does:
Mention broken yaml file when fed to mystery.py to find the offending player's file and their mistake faster

Add MultiMystery.py - documentation and config at head of file. Main objective is to streamline a multiworld mystery with individual settings.

Add HintedMultiServer.py, which adds a /hint command, such that /hint 1 Lamp broadcasts the location of player 1's Lamp to every client. This is to try and resolve "BK" without giving out or digging into the Spoiler manually, while being transparent to players as to what gets looked up.

### CODE REVIEW

**Comment by tcprescott** [comment]
Why make a copy of MultiServer.py?  Seems like a huge violation of DRY.

**Comment by Berserker66** [comment]
Because I wanted an untouched version if I needed it. This is not really meant to be merged without changes. Which is to say, let me know what you want with which modifications and I can probably make a pull request with just that.

**Comment by CaitSith2** [comment]
Well, this could very well be merged such that it has to be enabled on the command line options, and if its not enabled, `/hint ` does nothing.

File: Mystery.py [hunk 1]
```diff
@@ -65,14 +65,17 @@ def main():
...
+                except:
+                    raise ValueError(f"File {path} is destroyed. Please fix your yaml.")
             print(f"P{player} Weights: {path} >> {weights_cache[path]['description']}")
 
     erargs = parse_arguments(['--multi', str(args.multi)])
     erargs.seed = seed
     erargs.names = args.names
     erargs.create_spoiler = args.create_spoiler
-    erargs.race = True
+    erargs.race = False
```
Comment (CaitSith2) [inline Mystery.py]: Also suggested, make this hard-coded line into a command line argument that defaults to True if not present.

**Comment by Bonta0** [comment]
...

**Comment by Ijwu** [comment]
@Bonta0 Did you have a stroke over your keyboard or did you intend to contribute something here?

**Pull request closed without merging**